app/models/user.rb

```diff
@@ -1,29 +1,9 @@
 class User < ApplicationRecord
-  include Devise::JWT::RevocationStrategies::JTIMatcher
```

Comment:
User is unable to `seed` users data. Kindly see the below error.
![seed](https://user-images.githubusercontent.com/35504149/148553341-24d4db9c-3965-4e96-b7ae-58a44ed3214a.png)